core/test/unit/models/base/base_spec.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var should = require('should'),
+    sinon = require('sinon'),
+    _ = require('lodash'),
+    models = require('../../../../server/models'),
+    ghostBookshelf,
+    testUtils = require('../../../utils'),
+
+    sandbox = sinon.sandbox.create();
+
+describe.skip('Models: base', function () {
+    before(function () {
+        models.init();
+    });
+
+    afterEach(function () {
+        sandbox.restore();
+    });
+
+    describe('setEmptyValuesToNull', function () {
+        let knexMock;
+
+        before(function () {
+            knexMock = new testUtils.mocks.knex();
+            knexMock.mock();
+        });
+
+        after(function () {
+            knexMock.unmock();
+        });
+
+        // TODO: mock the base model und run those tests
```

Comment:
@kirrg001 I am done with the rest, except for this. I couldn't figure out how to test the base model functionality... But the functionality itself is tested within the post, user, and tags models.

Reply:
pushed a commit 👍